diff/schema_list_diff.go

```diff
@@ -158,14 +163,19 @@ func getGroupDifference(config *Config, state *state, schemaRefs1, schemaRefs2 o
 		}
 
 		if found, index2, err := findIndenticalSchema(config, state, schemaRef1, schemaRefs2, matched, filter); err != nil {
-			return nil, err
+			return nil, nil, err
 		} else if !found {
-			notContained = append(notContained, index1)
+			notContainedIdx = append(notContainedIdx, index1)
+			schemaName := schemaRef1.Ref
+			if schemaName == "" {
```

Comment:
Note that this condition will always be true because the function is only called for inline schemas.
The code would be cleaner if we could avoid this condition, but I'm not sure how to do it, perhaps you can think of a solution.

Reply:
you're right, it's only for inline schemas.
I defaulted to using the inlineSchemaPrefix (BaseSchema/RevisionSchema) and also renamed those function to getGroupDiffForInlineSchemas to make it clear that it's for inline schemas
wdyt?